Server/JokesOnYou.Web.Api/Repositories/TagRepository.cs

```diff
@@ -23,7 +23,9 @@ public TagRepository(DataContext context, IMapper mapper)
             _mapper = mapper;
         }
         public async Task<IEnumerable<TagReplyDto>> GetAllTagDtosAsync() => await _context.Tags.ProjectTo<TagReplyDto>(_mapper.ConfigurationProvider).ToListAsync();
+        public async Task<TagReplyDto> GetTagDtoAsync(int id) => await _context.Tags.ProjectTo<TagReplyDto>(_mapper.ConfigurationProvider).FirstOrDefaultAsync(t => t.Id == id);
```

Comment:
This line is 177 cols long. Split it in to 2 lines so its more readable.

Reply:
@chrisK00, done!